daemon/networkdriver/bridge/driver.go

```diff
@@ -140,6 +194,16 @@ func InitDriver(job *engine.Job) engine.Status {
 		if err := ioutil.WriteFile("/proc/sys/net/ipv4/ip_forward", []byte{'1', '\n'}, 0644); err != nil {
 			job.Logf("WARNING: unable to enable IPv4 forwarding: %s\n", err)
 		}
+
+		if enableIPv6 {
+			// Enable IPv6 forwarding
+			if err := ioutil.WriteFile("/proc/sys/net/ipv6/conf/default/forwarding", []byte{'1', '\n'}, 0644); err != nil {
+				job.Logf("WARNING: unable to enable IPv6 default forwarding: %s\n", err)
+			}
+			if err := ioutil.WriteFile("/proc/sys/net/ipv6/conf/all/forwarding", []byte{'1', '\n'}, 0644); err != nil {
```

Comment:
Ok, so let me paint this picture a different way (from the perspective of a prospective user):

1. user has IPv6; they don't like configuration hassle, so they go the easy route and setup RA
2. user now goes to http://whatismyv6.com and sees IPv6, knows they have IPv6, are happy :tada:
3. user starts using Docker and thinks "wow, this is cool, I wish it could use my IPv6!"
4. user notices that it has a toggle on the daemon to enable IPv6, so of course they go turn it on immediately, and they even hand it a `fixed-cidr` because they have one -- it's the one their router is handing out via RA
5. user no longer has IPv6 on their computer thanks to RA being disabled by Docker